src/Toolkit/Toolkit.Maui/FloorFilter/FloorFilter.cs

```diff
@@ -146,7 +146,11 @@ internal async void NavigateForward(ContentPage page)
     {
         try
         {
+#if WINDOWS && ANDROID
```

Comment:
Please add link to maui issue and explain here why we do this different. Same below.